.env

```diff
@@ -34,5 +34,7 @@ CONTENT_PREVIEW_MODE=true
 # Rollbar (only if you want to test errors are being reported locally)
 # ROLLBAR_ACCESS_TOKEN="..."
 # ROLLBAR_ENV=dev
+# VUE_APP=ROLLBAR_ACCESS_TOKEN="..."
```

Comment:
This doesn't look quite right to me?

Also, is it possible to reuse the same access token for the client side error reporting, or does Rollbar require a separate access token?

Reply:
Typo good spot!

Rollbar needs two different tokens, one for client & one for server! Updated this file to be more explicit.